src/neptune/new/types/atoms/file.py

```diff
@@ -139,19 +115,15 @@ def from_content(content: Union[str, bytes], extension: Optional[str] = None) ->
         .. _from_content docs page:
            https://docs.neptune.ai/api/field_types#from_content
         """
-        if isinstance(content, str):
-            ext = "txt"
-            content = content.encode("utf-8")
-        else:
-            ext = "bin"
-
-        if limits.file_size_exceeds_limit(len(content)):
-            content = b""
+        verify_type("content", content, (bytes, str, type(None)))
+        verify_type("extension", extension, (str, type(None)))
 
-        return File(content=content, extension=extension or ext)
+        file_composite = _InMemoryComposite(content, extension)
+        return File(file_composite=file_composite)
 
     @staticmethod
     def from_stream(stream: IOBase, seek: Optional[int] = 0, extension: Optional[str] = None) -> "File":
+        # TODO: docs (update)
```

Comment:
We could inform user that from now on, he will be able to use file created with `File.from_stream` at most once.

Reply:
@normandy7